### PR TITLE
Add additional indexed fields from MODS for increased parity with existing public_dc indexing

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Comment by cbeer** [comment]
I suspect any such test would be pretty brittle, and is mostly testing if stanford-mods works (which we oughtn't need to do here..). Happy to be persuaded, but I'm not sure "nice" is emphatic enough.

File: lib/dor/datastreams/desc_metadata_ds.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ class DescMetadataDS < ActiveFedora::OmDatastream
       t.scale       :index_as => [:symbol]
       t.topic       :index_as => [:symbol, :stored_searchable]
       t.abstract    :index_as => [:stored_searchable]
+      t.identifier  :index_as => [:symbol, :stored_searchable]
```
Comment (drh-stanford) [inline lib/dor/datastreams/desc_metadata_ds.rb]: is this used anywhere? or is it just for completeness?

Comment (cbeer) [inline lib/dor/datastreams/desc_metadata_ds.rb]: This is intended to eventually replace the `public_dc_identifier_tesim` field.

**Comment by jmartin-sul** [comment]
@cbeer:  yeah, i do agree that such tests might be brittle... i'm not sure that having expectations about what stanford-mods returns would be a bad thing, but i can also see how implementing/maintaining that could be more of a headache than it's worth.

anyway, merging...

**Pull request merged**